dashboard/app/mailers/ops_mailer.rb

```diff
@@ -34,9 +34,9 @@ def workshop_reminder(workshop, recipient)
     @workshop = workshop
     @recipient = recipient
 
-    subject = "Important: Your #{@workshop.phase_info[:long_name]} workshop is coming up in
```

Comment:
I'm cool with any of the approaches. I was more concerned with consistency: phase_long_name has its own safe method but :prerequisite_phase does not and requires the nil check on line 39. Why are they treated differently back to back?